pkg/scheduler/internal/cache/cache.go

```diff
@@ -261,7 +261,7 @@ func (cache *schedulerCache) UpdateNodeInfoSnapshot(nodeSnapshot *nodeinfosnapsh
 		cache.updateNodeInfoSnapshotList(nodeSnapshot, updateAllLists)
 	}
 
-	if len(nodeSnapshot.NodeInfoList) != len(nodeSnapshot.NodeInfoMap) {
+	if len(nodeSnapshot.NodeInfoList) != cache.nodeTree.numNodes {
 		errMsg := fmt.Sprintf("snapshot state is not consistent, length of NodeInfoList=%v not equal to length of NodeInfoMap=%v "+
```

Comment:
Need to update the error message, `NodeInforMap` -> `NodeTree`

Reply:
done.